packages/scoutgame/src/points/__tests__/calculateBuilderLevel.spec.ts

```diff
@@ -55,34 +59,39 @@ function validateCalculations({
 
   expect(builder42).toMatchObject<BuilderAggregateScore>({
     builderId: expect.any(String),
-    totalPoints: 3800,
-    averageGemsPerWeek: 1266,
+    totalPoints: 2500,
+    averageGemsPerWeek: 2500,
     centile: 80,
     level: 9,
-    firstActiveWeek: '2025-W08'
+    firstActiveWeek: '2025-W04'
   });
   expect(builder87).toMatchObject<BuilderAggregateScore>({
     builderId: expect.any(String),
-    totalPoints: 4300,
-    averageGemsPerWeek: 614,
+    totalPoints: 3000,
+    averageGemsPerWeek: 1500,
     centile: 57,
     level: 6,
-    firstActiveWeek: '2025-W04'
+    firstActiveWeek: '2025-W03'
   });
 
   expect(builder156).toMatchObject<BuilderAggregateScore>({
     builderId: expect.any(String),
-    totalPoints: 1000,
-    averageGemsPerWeek: 250,
+    totalPoints: 2000,
+    averageGemsPerWeek: 666,
     centile: 23,
     level: 3,
-    firstActiveWeek: '2025-W07'
+    firstActiveWeek: '2025-W02'
   });
 }
 
-describe('calculateBuilderLevels', () => {
+describe.skip('calculateBuilderLevels', () => {
```

Comment:
just fyi - is it intentional to skip this? r u planning on fixing later?

Reply:
Fixed what caused tests to interfere with each other, unskipped